circuits/ts/__tests__/utils.ts

```diff
@@ -26,7 +26,4 @@ async function getSignal(circuit: CircuitT, witness: bigint[], name: string): Pr
     return BigInt(witness[indexInWitness]);
 }
 
-export {
-    str2BigInt,
-    getSignal
-}
+export const STATE_TREE_DEPTH = 10
```

Comment:
Is there a specific reason this is here? Perhaps it could fit well in `constants.ts`

Reply:
totally